plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/data/transformers/ArrayAttributeTransformer.java

```diff
@@ -48,8 +49,8 @@ public void transformAttribute(@NotNull DBCSession session, @NotNull DBDAttribut
         }
         if (collectionType != null) {
             DBSDataType componentType = collectionType.getComponentType(session.getProgressMonitor());
-            if (componentType instanceof DBSEntity) {
-                ComplexTypeAttributeTransformer.createNestedTypeBindings(session, attribute, rows, (DBSEntity) componentType);
+            if (componentType instanceof DBSEntity || componentType instanceof DBSBindableDataType) {
```

Comment:
I'm embarrassed that only "Entity" classes was used before, but we also use "DataType" now.
Maybe DBSBindableDataType must also be entity.

Reply:
also looks strange that for "normal" columns we use PostgreDataType (entity), but for composite PostgreDataTypeAttribute (not entity)

Reply:
This check is not used here now as PostgreDataType is DBSEntity and we work only in PostgreSQL now. So, I removed it.